pkg/k8shandler/namespace.go

```diff
@@ -0,0 +1,88 @@
+package k8shandler
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openshift/elasticsearch-operator/pkg/utils"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/retry"
+)
+
+const (
+	namespaceNameEnvVar  = "POD_NAMESPACE"
```

Comment:
To stay compatible with the terminology of operators-sdk as we are due to upgrade (see #291), if this is the namespace of the operator, it's better to call it `OPERATOR_NAMESPACE`. If it's the namespace of the owned resources e.g. elasticsearch pods then better to call it `WATCH_NAMESPACE`.

Reply:
this is following the pattern of the required env var `POD_NAME` which comes from the same metadata. `WATCH_NAMESPACE` is a different since for EO we watch it all namespaces, this is specific to the namespace the operator pod is running in.